front/app/components/admin/ContentBuilder/Widgets/AboutBox/index.tsx

```diff
@@ -6,11 +6,30 @@ import { Box } from '@citizenlab/cl2-component-library';
 
 // i18n
 import messages from './messages';
+import { useNode } from '@craftjs/core';
+import useCraftComponentDefaultPadding from '../../useCraftComponentDefaultPadding';
+import { useParams } from 'react-router-dom';
+import useProjectBySlug from 'api/projects/useProjectBySlug';
 
-const AboutBox = ({ projectId }: { projectId: string }) => {
+const AboutBox = () => {
+  const { projectId, slug } = useParams() as {
+    projectId: string;
+    slug: string;
+  };
+  const { data: project } = useProjectBySlug(slug);
+  const projectID = projectId || project?.data.id;
+  const { parent } = useNode((node) => ({
+    parent: node.data.parent,
+  }));
+  const componentDefaultPadding = useCraftComponentDefaultPadding(parent);
```

Comment:
I made a hook for determining the default padding as to not repeat it in each component.

Reply:
No reason not to do it, except that I didn't think of it for some reason. Much better, indeed. Let's keep the name as is in my PR, I feel it's more descriptive and clear, and I'll move the `useNode` inside the hook. This should take care of conflicts, at least to some extent 👍